src/reducer/helper/talks.ts

```diff
@@ -0,0 +1,75 @@
+//#region > Imports
+//> Models
+// Contains the repository and talk model of the database
+import { Repository, Talk } from "../database/models";
+//#endregion
+
+//#region > Interfaces
+/** @interface Talk defines the structure of a talk object */
+interface ITalk extends Talk {
+  /**
+   * Repository: The repository object which contains the talk.
+   */
+  repository: Repository;
+  /**
+   * //#LEGACY
+   * @todo Move this to snek-at/engine
+   * Uid: A temporary random string to identify a talk.
+   */
+  uid: string;
+}
+//#endregion
+
+//#region > Functions
+/**
+ * @function
+ * @param {number} length A number which represents the length of the unique id
+ * @returns A random unique id
+ * @description Generates a random unique id with a specific length
+ */
+function randomUid(length: number) {
+  let result = "";
+  let characters =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
The only possibility I see to use a Regex to generation a random Uuid would be to use
http://fent.github.io/randexp.js/.

But since this function is only used temporarily anyway, since the backend does not currently support a UID for talks, in our view it is not necessary to install an extra package for this. Therefore we use plain JS